gdnative-core/src/core_types/color.rs

```diff
@@ -38,6 +38,16 @@ impl Color {
         unsafe { (get_api().godot_color_get_v)(self.sys()) }
     }
 
+    #[inline]
+    pub fn lerp(&self, other: &Color, weight: f32) -> Color {
```

Comment:
`Color` is a value type that can be cheaply copied, thus `other` should be taken by value.